R/check_ebp_arguments.R

```diff
@@ -235,6 +244,30 @@ fw_check1 <- function(pop_data, mod_vars, pop_domains, smp_data,
     }
   }
 
+  if(is.null(aggregate_to) != TRUE){
+    if (!(aggregate_to %in% colnames(pop_data))) {
+      stop(paste0("The domain variable ", aggregate_to, " is not contained in
+                  pop_data. Please provide valid variable name for the
+                  area-level for aggregation."))
```

Comment:
I would delete the ""for the area-level"". Just: Please provide valid variable name for the aggregation level.